docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,506 @@
+
+### Show pop-up on resume
+
+In this section, the example explains how to get the timing of resuming event in the Web application. On the timing, a pop-up with `Hello. Resume occurred!!` message appears.
+
+Following is the **tmplDefault.js** JavaScripte file at `shared/res/`.
+
+![JavaScript file for template](media/addon_template_js_file.png)
+
+Following are the used events and its callback for addons in the code:
+
+| Used Event     | Matched Callback    |
+| -------------- | ------------------- |
+| lcPrelaunch | prelaunch |
+| lcResume | resume |
+| lcSuspend | suspend |
+| lcQuit | quit |
+
+For more information, see [basic events in addons](../../guides/applications/addon.md#event).
+
+In this file, each callback has only `console.log()` as implementation.
+
+To show a pop-up with a message, use `lcResume`. Remove all the lines related to other events except `lcResume`.
+
+![JavaScript file for resume](media/js_for_resume.png)
+
+`addonapi` includes APIs for addons. To show the pop-up, you can use `addonapi.window.showMessageBox()`.
+
+```
+const addonAPI = require('addonapi');
+
+resume(winId) {
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'Resume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+
+
+To verify whether the modified file works as expected, follow these steps:
+
+1. Using the changed file, build a signed package again.
+
+2. Before installing the newly built package, remove the installed addon. In WebApp Addon Setting App, to remove the installed addon, click **Uninstall** ![addon settings x button](media/addon_settings_x_btn.png).
+
+3. To install and run the newly built addon package, follow the [steps to run addon](#run-addon) again. 
+
+    The following pop-up apprears:
+
+    **Figure: Addon Pop-up on Resume Event in Basic UI**
+
+    ![Showing pop-up in Basic UI](media/hello_resume.png)
+
+
+<a name="load-html-from-addon"></a>
+### Load HTML file from addon
+
+In an addon, you can load your own HTML file. In this section, an example of an HTML file that shows `Hello, I'm an Addon` is explained.
+
+Ensure that all the files related to an addon are located under `shared/res` of your addon project. Therefore, the added HTML file must also be located in the same directory.
+
+To add a new HTML file for addon, expand the **BasicAddon** in **Project Explorer**. Then, right-click on the **res** folder and select **New > File**. Enter the file name that you want to use. In this example, **hello.html** is inserted as the file name.
+
+![Add a new file](media/new_file.png)
+
+![Generate the HTML file](media/generate_hello_html.png)
+
+Following is an example of **hello.html** file:
```

Comment:
Please add "the" before "**hello.html** file:".

Reply:
done